src/queries/vouchers.ts

```diff
@@ -5,12 +5,34 @@ import { VoucherLogin } from '../types';
 
 // TODO: memberId: 1 will be updated later. 1 will be replaced by the id of the logged in user
 export const GET_VOUCHERS = gql`
-  query Vouchers($limit: Int, $order: GenericItemOrder) {
-    vouchers(limit: $limit, order: $order) {
+  query GenericItems(
+    $ids: [ID]
+    $limit: Int
+    $offset: Int
+    $order: GenericItemOrder
+    $dataProvider: String
+    $categoryId: ID
+    $genericType: String
+  ) {
+    genericItems(
+      ids: $ids
+      limit: $limit
+      skip: $offset
+      order: $order
+      dataProvider: $dataProvider
+      categoryId: $categoryId
+      genericType: $genericType
```

Comment:
could this be fixed to `Voucher` instead of using a variable?

Reply:
Yes, that's absolutely right. Since this query is only Vouchers query, `Voucher` value can be used for `genericType`.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/5ea2c6f1c749f78bbf8a1c1f43924dc2fee84051